sources/Runner/GameScene.cpp

```diff
@@ -34,6 +36,10 @@ namespace BBM
 		scene->addEntity("camera")
 			.addComponent<PositionComponent>(8, 20, 7)
 			.addComponent<CameraComponent>(Vector3f(8, 0, 8));
+		scene->addEntity("Timer")
+			.addComponent<TimerComponent>(std::chrono::seconds(60), [](WAL::Entity &, WAL::Wal &) {
```

Comment:
60 seconds is a bit low, you should set it to 5 minutes I think.

Reply:
3 mins